CONTRIBUTING.md

```diff
@@ -218,33 +218,17 @@ Some things to keep in mind before publishing the release:
 
 ### Run a release...
 
-#### ...with automatically generated change logs
-
 1) Switch to `master` branch locally.
-2) Run 'git pull origin master'.
-3) Run `git fetch --all` to make sure all tags and commits are fetched.
-4) Run `melos version`. This will auto commit, update the changelogs using git commit messages from the last release, and also tag the release for versioning.
-5) Run `git push --follow-tags` to push the auto commits and tags to the remote repository.
+2) Run `git pull origin master`.
+3) Run `git pull --tags` to make sure all tags are fetched.
+4) Create new branch with the signature "release/[year]-[month]-[day]".
+5) Run `melos version --no-git-tag-version` to automatically version packages and update Changelogs.
 6) Run `melos publish` to dry run and confirm all packages are publishable.
-7) Run `melos publish --no-dry-run` to now publish to Pub.dev.
-8) Update the documentation by manually triggering the [publish docs workflow](https://github.com/FirebaseExtended/flutterfire/actions/workflows/docs.yaml); ![image](https://user-images.githubusercontent.com/5347038/117001552-69095780-acda-11eb-934c-a228be6f5c7a.png)
-
-
-#### ...with manual change log edits
-
-To run a release and manually edit the change log, do the following steps:
-
-1) Switch to `master` branch locally.
-2) Run 'git pull origin master'.
-3) Run `git fetch --all` to make sure all tags and commits are fetched.
-4) Run `melos version --no-git-tag-version`. This will skip auto commiting and git tagging, leaving your git tree dirty with version bumps and changelog entry changes.
-5) Update the `CHANGELOG.md` files that you manually want to edit/reword.
-6) Add and commit all the `CHANGELOG.md` and `pubspec.yaml` files that were modified by `melos version` (using the standard release commit message, e.g. `chore(release): publish packages`).
-7) `melos publish` to dry run and confirm all packages are publishable.
-8) `melos publish --no-dry-run --git-tag-version` to now publish to Pub.dev (`--git-tag-version` will add missing git tags since we skipped tagging in step 1).
-9) Push your changes to `master`; `git push --follow-tags`.
-10)Update the documentation by manually triggering the [publish docs workflow](https://github.com/FirebaseExtended/flutterfire/actions/workflows/docs.yaml); ![image](https://user-images.githubusercontent.com/5347038/117001552-69095780-acda-11eb-934c-a228be6f5c7a.png)
-
+7) After successful dry run, commit all changes with the signature "chore(release): prepare for release".
+8) Run `git push origin master` & open pull request for review on GitHub.
```

Comment:
Shouldn't be `git push origin master` here - we're pushing the new `release/*` branch instead